src/core_modules/capture-core/components/WidgetEnrollment/Actions/actions.types.ts

```diff
@@ -0,0 +1,42 @@
+import type { WithStyles } from '@material-ui/core';
+import type { UpdateEnrollmentOwnership } from './Transfer/hooks/useUpdateOwnership';
+
+type QueryRefetchFunction = any;
```

Comment:
QueryRefetchFunction is defined in `src/core_modules/capture-core-utils/types/app-runtime.ts`. You should import it and use it and not create a local type